MyTests/Tests/LoginValidationCases.cs

```diff
@@ -0,0 +1,35 @@
+﻿using MyObjects.Helpers;
+using MyObjects.Pages.Saucedemo;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MyTests.Tests
```

Comment:
did you manage to run different tests in parallel?
I'm referring to this task
2. Run your tests in parallel - choose level of parallelism of your preference.